voltixApp/VoltixApp/View Models/SendCryptoViewModel.swift

```diff
@@ -207,40 +194,40 @@ class SendCryptoViewModel: ObservableObject {
             }
             
         } else if tx.coin.chain.name.lowercased() == Chain.Ethereum.name.lowercased() {
-            let ethBalanceInWei = Int(eth.addressInfo?.ETH.rawBalance ?? "0") ?? 0 // it is in WEI
-            
-            if tx.coin.ticker.uppercased() == "ETH" {
-                if tx.totalEthTransactionCostWei > ethBalanceInWei {
-                    errorMessage = "walletBalanceExceededError"
-                    showAlert = true
-                    logger.log("Total transaction cost exceeds wallet balance.")
-                    isValidForm = false
-                }
-                
-            } else {
-                if let tokenInfo = eth.addressInfo?.tokens?.first(where: { $0.tokenInfo.symbol == tx.coin.ticker.uppercased() }) {
-                    print("tx.feeInWei \(tx.feeInWei)")
-                    print("ethBalanceInWei \(ethBalanceInWei)")
-                    
-                    print("has eth to pay the fee?  \(tx.feeInWei > ethBalanceInWei)")
-                    
-                    if tx.feeInWei > ethBalanceInWei {
-                        errorMessage = "mustHaveETHError"
-                        showAlert = true
-                        logger.log("You must have ETH in to send any TOKEN, so you can pay the fees. \n")
-                        isValidForm = false
-                    }
-                    
-                    let tokenBalance = Int(tokenInfo.rawBalance) ?? 0
-                    
-                    if tx.amountInTokenWei > tokenBalance {
-                        errorMessage = "walletBalanceExceededError"
-                        showAlert = true
-                        logger.log("Total transaction cost exceeds wallet balance.")
-                        isValidForm = false
-                    }
-                }
-            }
+//            let ethBalanceInWei = Int(eth.rawBalance) ?? 0// it is in WEI
```

Comment:
Please remove it if you don't need it anymore

Reply:
Done